Islandora/islandora_badges
# Add Crossref citations badge

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2038)

# What does this Pull Request do?
Adds a new Islandora Badges submodule, which generates citation counts via the Crossref REST API. This is a public API, without authentication, which would allow any Islandora site to get citation counts if they don't have access to Scopus or Web of Science credentials.

# What's new?
A new badge titled "Islandora Crossref Citations". Displays citation counts for objects with DOIs, using the public-access Crossref REST API. 

# How should this be tested?

* Enable the module and the block
* Add a few objects with and without valid  DOIs
* View the object, check the results

# Interested parties
@Islandora/7-x-1-x-committers  @bryjbrown @mjordan
